sdk/python/v1beta1/kubeflow/katib/models/v1beta1_trial.py

```diff
@@ -13,9 +13,8 @@
 import pprint
 import re  # noqa: F401
 
-import six
-
 from kubeflow.katib.configuration import Configuration
+import six
```

Comment:
The models files are auto-generated.

Reply:
Same as https://github.com/kubeflow/katib/pull/2242#discussion_r1677898994.